addons/im_livechat/static/src/embed/core/messaging_service_patch.js

```diff
@@ -4,9 +4,13 @@ import { Messaging } from "@mail/core/common/messaging_service";
 
 import { patch } from "@web/core/utils/patch";
 import { session } from "@web/session";
+import { SESSION_STATE } from "@im_livechat/embed/core/livechat_service";
 
 patch(Messaging.prototype, {
     initialize() {
+        if (this.env.services["im_livechat.livechat"].state === SESSION_STATE.PERSISTED) {
+            return super.initialize();
+        }
```

Comment:
This code is quite a maze!
In follow-up PR should clean code so there's no such things as
1. totally custom `Message.initialize()`
2. livechat service setup (lots of function calls should be reduced to very few)
3. `Message.initialize()` again, but now it's the good one

One solution would be to add support to pause `Messaging@initialize`, and move the custom code in livechat service. That way, the single point of entry is clear for livechat (livechat service), and it's clear `Messaging@initialize` happens afterwards.